### PR TITLE
[W-11455021] - Fixing ACH text for Experience Sites

### PR DESCRIPTION
**Pull request by gerardorodriguezsf**
# Changes
Removed spinner text related to Elevate when updating the payment method of an Elevate RD and the payment is ACH.

# Issues Closed
[W-11455021](https://gus.lightning.force.com/a07EE000011qCavYAE)

### CODE REVIEW

**Review by JoseRossiSF — APPROVED** [review]

File: force-app/main/default/lwc/rd2EntryForm/rd2EntryForm.js [hunk 1]
```diff
@@ -676,8 +677,8 @@ export default class rd2EntryForm extends LightningElement {
     async processCommitmentSubmit(allFields) {
         try {
             if (this.isElevateWidgetDisplayed()) {
-                this.loadingText = this.rd2Service.getPaymentProcessingMessage(this.rd2State.paymentMethod);
-
+                
+                this.loadingText = this.isExperienceSite && this.rd2Service.getPaymentProcessingMessage(this.rd2State.paymentMethod).includes(ELEVATE) ? '' : this.rd2Service.getPaymentProcessingMessage(this.rd2State.paymentMethod);
```
Comment (force2b) [inline force-app/main/default/lwc/rd2EntryForm/rd2EntryForm.js:681]: This feels fragile to me. The code is looking for the word "elevate" in text to determine if Elevate is enabled or not (I believe). Is there a better way to determine if Elevate is enabled or not? Assuming that donors cannot create a new Recurring Donation in this UI, then I believe the RD.CommitmentId__c field will have a value if it's an Elevate RD. Would that be a more reliable check to determine if Elevate is enabled?

Comment (gerardorodriguezsf) [reply force-app/main/default/lwc/rd2EntryForm/rd2EntryForm.js:681]: We should not show the word 'elevate' in the spinners if the component is used from the experience sites. i was checking the custom label value, but I agree it's a better idea to check the RD.CommitmentId__c. Let me apply the update.

Comment (force2b) [reply force-app/main/default/lwc/rd2EntryForm/rd2EntryForm.js:681]: @gerardorodriguezsf  Thanks. Is the intent to display nothing at all if "Elevate" is anywhere in the PaymentProcessing message?

Comment (gerardorodriguezsf) [reply force-app/main/default/lwc/rd2EntryForm/rd2EntryForm.js]: It' showing 'elevate' word only for ACH payments. I updated the condition to check the payment method.

**Review by force2b — APPROVED** [review]

**Pull request merged**